### PR TITLE
Coq 8.6 packages for plugin-utils, ext-lib, and charge-core

### PR DESCRIPTION
**Pull request by gmalecha**

### CODE REVIEW

File: released/packages/coq-plugin-utils/coq-plugin-utils.1.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+homepage: "https://github.com/gmalecha/coq-plugin-utils"
+bug-reports: "https://github.com/gmalecha/coq-plugin-utils/issues"
+license: "MIT"
+build: [
+  [make "-j%{jobs}%"]
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/PluginUtils"]
+depends: [
+  "coq" {>= "8.6" < "8.7"}
```
Comment (gares) [inline released/packages/coq-plugin-utils/coq-plugin-utils.1.3.0/opam]: should be `<8.7~`

**Comment by gares** [comment]
Would you mind making the descriptions a bit longer? They are also used to generate the Coq Package Index pages, and the ones of extlib or ChargeCore are very short.

**Comment by gares** [comment]
Thanks for the update.

I did not spot it before but the version thing should be `  "coq" {>= "8.5" & < "8.6~"}` (you miss the `&` symbol)

**Comment by gmalecha** [comment]
@gares I think this fixes the issue. I also added a max version to mirror-core because it doesn't build with 8.6 right now (plugin issues that I need to look into more).

**Pull request merged**